code/index.html

```diff
@@ -1,21 +1,46 @@
 <!DOCTYPE html>
-<html>
-
+<html lang="de">
+ 
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project Name</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>Carole & Tobias</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
     <!-- If you're using google fonts, that should be added here -->
+    <link href="https://fonts.googleapis.com/css2?family=Mukta:wght@300&family=Playfair+Display:wght@500&display=swap" rel="stylesheet">  
   </head>
+
   <body>
-    <!-- 
-      Write your code here inside the body to make it appear in the browser. 
-      Below is an example on how to use an image tag. Copy it and add it in the body, but outside the comment tag if you want to see how it looks like on the website. Then you can change the size and other styling for it in the css file.
+   
+    <img src="Hanging Flowers.png" alt="Flowers" class="image1">
```

Comment:
To be safe, we should always rename our files so that they don't have any spaces in the names. Dashes are fine though so a suggestion would be:
hanging-flowers.png